docs/Governance.md

```diff
@@ -58,13 +58,15 @@ Responsibilities:
   announce that fact in the [Slack channel](https://cilium.slack.com/messages/ebpf-for-windows)
   and [github discussion thread](https://github.com/microsoft/ebpf-for-windows/discussions/427).
 
-### Maintainer
-
-A maintainer can also merge pull requests.
-This corresponds to the "Write" role in github.
+### Maintainers
+This corresponds to the "Maintain" role in github. A maintainer can manage repository settings.
```

Comment:
I'd like to discuss this in the meeting on Monday.

Reply:
While I have restored the original roles and updated the `codeowner` file, let us discuss if we should have separate `maintainer` and `active contributor` roles.